packages/connect-popup/src/index.tsx

```diff
@@ -320,6 +320,22 @@ const handleMessageInCoreMode = (
                 info: method.info,
             });
             reactEventBus.dispatch({ type: 'state-update', payload: getState() });
+
+            const { settings } = getState();
+            const transport = core.getTransportInfo();
+            analytics.report({
+                type: EventType.AppReady,
```

Comment:
IFRAME.CALL sounds weird but it makes sense, there is, right now, one iframe.call per popup open, so probably good enough 🤷‍♂️